src/LCT.Services/Sw360Service.cs

```diff
@@ -61,7 +61,8 @@ public async Task<List<Components>> GetAvailableReleasesInSw360(List<Components>
                 string responseBody = await m_SW360ApiCommunicationFacade.GetReleases();
                 if (string.IsNullOrWhiteSpace(responseBody))
```

Comment:
not required , since we have a check after reserializing

Reply:
Removed